aider/main.py

```diff
@@ -326,7 +326,6 @@ def generate_search_path_list(default_file, git_root, command_line_file):
     uniq.reverse()
     files = uniq
     files = list(map(str, files))
-    files = list(dict.fromkeys(files))
```

Comment:
I'm pretty sure this doesn't actually change anything (maybe it was from a previous version to make the list unique).  If it actually does something, I'd love to know what.